scripts/src/public/services/offer_contents.ts

```diff
@@ -37,15 +38,111 @@ export const kikPoll: Poll = {
 		title: "Who do you primarily chat with on Kik?",
 		description: `Complete the poll to earn <span style='color:#047cfc;'>4,000</span> Kin`,
 		question: {
-			id: "favourite_color",
+			id: "who34",
 			answers: [
 				"New friends I've met on Kik",
 				"Friends I've met elsewhere online",
 				"Friends I know from real life",
 				"An equal mix of all three!",
 			],
 		},
-	}],
+	}, {
+		type: PageType.FullPageMultiChoice,
+		title: "What do you like to use the most when chatting with others on Kik?",
+		description: `Complete the poll to earn <span style='color:#047cfc;'>4,000</span> Kin`,
+		question: {
+			id: "what324",
+			answers: [
+				"Video chat", "Stickers", "Gifs", "Emojis", "Kik custom emojis", "Just regular text messages",
```

Comment:
When are we changing the answers to `id, value`?

Reply:
I know, We need to do it.